rskj-core/src/main/java/org/ethereum/util/RLP.java

```diff
@@ -71,6 +77,8 @@
  * @since 01.04.2014
  */
 public class RLP {
+    private static final Logger logger = LoggerFactory.getLogger(RLP.class);
```

Comment:
not sure we really need one more logger in this class.. RLP seems to be kinda data entity, like String class. Maybe FrameCodec is a better place for logging this?

Reply:
This change was made due to a SonarQube complain, throwing back the error is not an option here because some methods needs to return somethnig back instead of throwing the error

Reply:
hmm, so SonarQube requires the RLP class to have a logger?